AsyncDisplayKit/ASMapNode.mm

```diff
@@ -266,16 +277,17 @@ - (void)addLiveMap
     [weakSelf setNeedsLayout];
     [weakSelf.view addSubview:_mapView];
     
-    if (CLLocationCoordinate2DIsValid(_centerCoordinateOfMap)) {
-      [_mapView setCenterCoordinate:_centerCoordinateOfMap];
-    }
+//    if (CLLocationCoordinate2DIsValid(_centerCoordinateOfMap)) {
+//      [_mapView setCenterCoordinate:_centerCoordinateOfMap];
```

Comment:
I removed the line in removeLineMap regarding the centerCoordinate because it was causing the map to drift, I didn't actually try it with this line intact, besides, shouldn't ` [weakSelf applySnapshotOptions];` take care of that, since we're setting the region inside that method?